modulo_1/Página Guilherme Martins/main.js

```diff
@@ -0,0 +1,94 @@
+//document.querySelector('html').onclick = function() {
+    //alert('Ai! Pare de me cutucar!');
+//}
+
+const project_name = [];
+const lngg = [];
+const vsblt = [];
+const webadd = [];
+
+let myimg = document.querySelector('img');
+myimg.onclick = function() {
+	let mysrc = myimg.getAttribute('src');
```

Comment:
Nit: Não necessariamente você precia utilizar o getAttribute.
Pode utilizar diretamente myimg.src

Como você está usando uma função de escopo fechado, você pode usar this ao invés de myimg.

Se você estivesse usando um escopo aberto como:
```javascript
myimg.onclick = () => {

}
```
o this seria o document